src/common/latest/sonarqube/Endpoint.ts

```diff
@@ -67,14 +68,10 @@ export default class Endpoint {
     const proxyUrl = getProxyFromURI(new URL(endpointUrl))
       ? getProxyFromURI(new URL(endpointUrl))
       : undefined;
-    if (proxyUrl) {
-      tl.debug("Using proxy agent from environment: " + proxyUrl);
-    } else {
-      tl.debug("Not using a proxy agent");
-    }
```

Comment:
Dropped and moved to line `74`

Reply:
I guess there's no need to log when the proxy agent isn't used? 

only reason I could see is calling out to a user when they believe PROXY is used, but indeed we skip it

Reply:
My thinking was, we can still know because of the presence/absence of this debug log? Also found it annoying to have this log on every request, but idk, wdyt?

Reply:
that makes sense, reduce the noise, and if the user sets the proxy they should see something about it, if not it's not configured properly 👍🏻